scapy/contrib/opc_da.py

```diff
@@ -272,10 +272,10 @@ class LenStringPacket(Packet):
     name = "len string packet"
     fields_desc = [
         FieldLenField('length', 0, length_of='data', fmt="H"),
-        ConditionalField(StrLenField('data', None,
+        ConditionalField(StrLenField('data1', None,
                          length_from=lambda pkt:pkt.length + 2),
                          lambda pkt:pkt.length == 0),
-        ConditionalField(StrLenField('data', '',
+        ConditionalField(StrLenField('data2', '',
                          length_from=lambda pkt:pkt.length),
                          lambda pkt:pkt.length != 0),
```

Comment:
You would typically use a `MultipleTypeField` to keep a unique `data` name for both cases. See `help(MultipleTypeField)`. Also the first field could be replaced by `StrLenField('data', b'', length=2)` instead of using the lambda and `length_from`.